scan-task.ts

```diff
@@ -55,6 +58,11 @@ crawler.on('fetchcomplete', (queueItem, responseBuffer, response) => {
 });
 crawler.on('complete', function () {
   console.log('Scan complete');
+  if (!hasFoundAnyPages) {
+    console.error('No pages were found while crawling site; exiting');
```

Comment:
Do you think we can make this a bit more helpful? I _think_ it's true that the only time this can happen is when the user puts in a bad URL. Even a site with only one page would return one result. So I think it might be safe to suggest here that the user might have made a mistake with the URL. What do you think? There's nothing untrue about the current message, but it seems a little bit vague/opaque for what might be a pretty specific situation. 

Suggestion: `No pages were found for this site. The two most likely reasons for this are: 1) the URL is incorrect or 2) the crawler is being denied by a robots.txt file.` 

Thoughts?

Reply:
Updated!